Automation/General/qama and qawa.md

```diff
@@ -0,0 +1,35 @@
+> All code is guilty, until proven innocent.
+
+`qama` and `qawa` are our **template projects** for test automation.
+
+The idea behind them is to help you quickly start writing maintainable and easy to understand UI tests.
+While **qama** is designed to work primarily with mobile, **qawa** focuses on web projects.
+
+
+## Project structure
+
+Both projects are structured using the _page object model_.
+
+In short, that means that:
+
+- Locators are placed in a page class.
+- Tests are placed in a test class.
+
+Read [this page object model article](https://martinfowler.com/bliki/PageObject.html) for more information.
```

Comment:
Do you think Selenium docs on [page object models](https://www.selenium.dev/documentation/test_practices/encouraged/page_object_models/) would be better here?